deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/CloudResourcesTable.tsx

```diff
@@ -101,27 +101,27 @@ const FILTER_SEARCHPARAMS: Record<string, string> = {
   azure_account_ids: 'Azure account',
 };
 
-const CLOUD_PROVIDERS = [
-  {
-    label: 'AWS',
-    value: 'aws',
-  },
-  {
-    label: 'GCP',
-    value: 'gcp',
-  },
-  {
-    label: 'Azure',
-    value: 'azure',
-  },
-];
-
 const getAppliedFiltersCount = (searchParams: URLSearchParams) => {
   return Object.keys(FILTER_SEARCHPARAMS).reduce((prev, curr) => {
     return prev + searchParams.getAll(curr).length;
   }, 0);
 };
 
+function getCloudResourcePrettyName(provider: ModelCloudResourceCloudProviderEnum) {
```

Comment:
`getCloudProviderPrettyName` would be a better name

Reply:
fixed